pxtlib/main.ts

```diff
@@ -389,6 +389,7 @@ namespace pxt {
         authcodeUrl?: string; // "/beta---authcode"
         isStatic?: boolean;
         verprefix?: string; // "v1"
+        isChinaIp?: boolean;
```

Comment:
this looks unrelated?

Reply:
Ah, fair point. This was a way for me to put up incremental progress from hiding video (and/or remember where that file was), but maybe not a good idea to put mix multiple work streams in one pull request.

Reply:
Thanks for feedback!

Reply:
Ah, yes, typically best to just keep a different branch and if you get a small portion done you can even put up a 'draft' pr -- if you leave traces of separate work in a PR, it can be hard to track later and/or cause issues if a PR gets put on the backlog for some reason -- e.g. you find out you have to test on a new device and wait for that to arrive, then the next PR might be blocked waiting for the previous one to be merged as it's missing some code or something, or when someone is reviewing the history they might miss a seemingly unrelated pr adding a crucial feature.

(these things are not likely to occur with one line changes like this as much, but the habit is more important -- it's easy for one line changes to become functions or small features if you're not paying attention :) )

Reply:
thanks for explanation, have removed these changes!